backend/tests/conftest.py

```diff
@@ -698,7 +698,6 @@ async def animal_person_schema_unregistered(db: InfrahubDatabase, node_group_sch
                 "attributes": [
                     {"name": "name", "kind": "Text", "unique": True},
                     {"name": "height", "kind": "Number", "optional": True},
-                    {"name": "bag", "kind": "Text", "optional": True, "unique": True},
```

Comment:
With the fix, `bag` becomes required if we create more than one `TestPerson`, thus I introduced `person_schema_unique_attr_non_hfid_unregistered` that `test_non_unique_value_raises_error` can now uses, so existing tests relying on this schema are not impacted.